lib/Cargo.toml

```diff
@@ -5,6 +5,7 @@ edition = "2021"
 rust-version = "1.62.1"
 
 [dependencies]
+serde = { version = "1.0", features = ["derive"] }
```

Comment:
Need to specify the version to be `1.0.103`, anything lower than that would not compile properly.